src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -97,10 +98,12 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
     if (encoding_supports_data_type(encoding_config.default_encoding_spec.encoding_type, column_data_type)) {
       chunk_encoding_spec.push_back(encoding_config.default_encoding_spec);
     } else {
-      std::cout << " - Column '" << table_name << "." << table->column_name(column_id) << "' of type ";
-      std::cout << column_data_type << " cannot be encoded as ";
-      std::cout << encoding_config.default_encoding_spec.encoding_type << " and is ";
-      std::cout << "left Unencoded." << std::endl;
+      // Use an osyncstream to prevent multiple threads writing at the same time to std::cout.
+      auto ostream = std::basic_osyncstream(std::cout);
+      ostream << " - Column '" << table_name << "." << table->column_name(column_id) << "' of type ";
+      ostream << column_data_type << " cannot be encoded as ";
+      ostream << encoding_config.default_encoding_spec.encoding_type << " and is ";
+      ostream << "left Unencoded." << std::endl;
```

Comment:
Can you add a short example of what you are fixing with this PR and how the result looks?

Reply:
Ah, I see. Updated it.